wiki/Game_modifier/No_mod/en.md

```diff
@@ -0,0 +1,29 @@
+---
+stub: true
+tags:
+  - no mod
+  - nomod
+  - no-mod
+  - NM
+  - normal
+  - normal mod
+---
+
+<!-- TODO:
+- add more info regarding usage in tournaments. preferably formatted as: ## Usage | ### Tournaments | ### Community -->
+
+# No mod (term)
```

Comment:
Is there a reason why this a "term" but Free mod is a "tournament category"?

Reply:
`No mod` would prob be classified as a term, because there's so much "I'm good at no mod. I'm terrible at no mod" among non-tournament players. Whereas `NoMod` would be considered a tournament category. Not sure if these should be two separate stubs tho.